package.json

```diff
@@ -46,47 +45,40 @@
   },
   "homepage": "https://github.com/statelyai/xstate#readme",
   "dependencies": {
-    "@actions/exec": "^1.0.2",
     "@babel/core": "^7.23.3",
     "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/preset-env": "^7.23.3",
     "@babel/preset-react": "^7.23.3",
     "@babel/preset-typescript": "^7.23.3",
     "@changesets/changelog-github": "^0.4.8",
     "@changesets/cli": "^2.26.2",
+    "@jest/types": "^29.6.3",
     "@manypkg/cli": "^0.16.1",
     "@manypkg/get-packages": "^1.1.3",
     "@preconstruct/cli": "^2.8.1",
-    "@testing-library/jest-dom": "^6.1.4",
     "@testing-library/vue": "^6.6.1",
     "@types/jest": "^29.5.10",
     "@types/node": "^12.11.1",
     "@vue/compiler-sfc": "^3.0.11",
     "@vue/vue3-jest": "^29.2.6",
     "babel-jest": "^29.7.0",
     "babel-preset-solid": "^1.8.4",
-    "gh-pages": "^2.0.1",
     "husky": "^3.1.0",
     "jest": "^29.7.0",
     "jest-config": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
-    "jest-regex-util": "^29.6.3",
+    "jest-resolve": "^29.7.0",
+    "jest-snapshot": "^29.7.0",
     "jest-watch-typeahead": "^2.2.2",
+    "knip": "https://pkg.pr.new/knip@bd16c98",
```

Comment:
I subscribed to `knip` release notifications so I can switch this to an actual version once it's published

Reply:
I'd prefer to block this PR on that release. I had unpleasant experiences in the past with such alternative registered - mainly CodeSandbox CI just decided to forget a package that was published using that service.

Reply:
Absolutely.

Reply:
> I'd prefer to block this PR on that release. I had unpleasant experiences in the past with such alternative registered - mainly CodeSandbox CI just decided to forget a package that was published using that service.

Damn you, Codesandbox! ✊☁️